pkgs/applications/editors/emacs/elisp-packages/update-melpa.el

```diff
@@ -257,10 +257,8 @@ return Promise to resolve in that process."
       url (lambda (status)
             (funcall resolve (condition-case err
                                  (progn
-                                   (goto-char (point-min))
-                                   (search-forward "\n\n")
-                                   (message (buffer-substring (point-min) (point)))
-                                   (delete-region (point-min) (point))
+                                   (message "%S" (url-http-parse-headers))
+                                   (goto-char url-http-end-of-headers)
```

Comment:
> fix an compatiblility issue with emacs 29 and recent melpa update

Could you explain what the issue is and how this change fixes that issue?

Reply:
It is wired. `(search-forword "\n\n")` failed on the day I tested, but it works today. It is not an issue caused by emacs 29, but something on melpa server probably.
IMO, url-http maintained by emacs itself is better than a manual parser here. I edited the pr title and commit message.